browse/static/js/plaudit.js

```diff
@@ -0,0 +1,49 @@
+(async function () {
+  $output = $('#plauditContainer');
+  const articleDoi = $output[0].dataset.doi;
+  const articleTitleElementContent = document
+    .querySelector("h1.title")
+    .textContent;
+  const articleTitle = articleTitleElementContent.startsWith("Title:")
+    // The `h1.title` has a `span.description` preceding the title that just
+    // says "Title:", so we'll need to strip that off:
+    ? articleTitleElementContent.substring("Title:".length)
+    : articleTitleElementContent;
+
+  const endorsementsResponse = await fetch(
+    `https://api.eventdata.crossref.org/v1/events?mailto=arxiv_integration@plaudit.pub&obj-id=${encodeURIComponent(`https://doi.org/${articleDoi}`)}&source=plaudit`
```

Comment:
Wanted to call this out: the public endorsement data is now fetched from CrossRef's Event Data API, which accepts an email address to contact when something goes wrong. I currently set that to `arxiv_integration@plaudit.pub`, but let me know if you'd prefer that to be an email address under arXiv's control (and if so, which?).

Reply:
I think it's OK if you look into any issues first. If it's something on our end, you can then let us know, and we'll be as responsive as usual :-).